### PR TITLE
age_gender_hsct20_ni

### PR DESCRIPTION
**Pull request by niuni02**
Completed the same tasks to find demographic data for Northern Ireland but from a health view (Health and Social Care Trusts). Manipulated the data to group age and gender. Added roxygen comments in R/data.R and then built documentation in man/.

### CODE REVIEW

**Comment by MikeJohnPage** [comment]
It looks like you created this branch off the old branch from PR #7 which has just been merged. I would consider always creating new branches off main for the simple GitHub flow model we are using as a general rule, unless you have good reason to do so otherwise.

File: data-raw/age_gender_hsct20_ni.R [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+# ---- Load internal sysdata.rda file with URL's ----
+load_all(".")
+
+# ---- Download data ----
+query_url <-
+  query_urls |>
+  filter(id == "age_gender_hsct20_ni") |>
+  pull(query)
+
+download <- tempfile(fileext = ".xlsx")
```
Comment (MikeJohnPage) [inline data-raw/age_gender_hsct20_ni.R:19]: Please change extension to `".ods"`

File: R/data.R [hunk 2]
```diff
@@ -1,3 +1,24 @@
+#' Age and gender grouped data for Northern Ireland from Population 2020 (Health and Social Care Trusts)
+#' 
+#' A dataset containing Population 2020 age and gender group count for each HSCT in Northern Ireland.
```
Comment (MikeJohnPage) [inline R/data.R:3]: Please wrap lines at 80 characters width (if you use RStudio you can add a vertical line in the editor as a guide).

File: R/data.R [hunk 3]
```diff
@@ -1,3 +1,24 @@
...
+#' 
+#' @format A data frame of class "tbl" with 5 rows and 11 variables:
+#' \describe{
+#'   \item{hsct20_name}{Health and Social Care Trust name}
+#'   \item{hsct20_code}{Health and Social Care Trust code}
+#'   \item{total_population}{Total population in NI}
+#'   \item{total_female_population}{Total female population in NI}
+#'   \item{total_male_population}{Total male population in NI}
```
Comment (MikeJohnPage) [inline R/data.R:11]: Can we remove the "in NI" suffix to these descriptions to clean them up a little?

File: R/data.R (hunk 2 above)
Comment (MikeJohnPage) [reply R/data.R:3]: Also please remove the word "group", as I don't think it adds anything.

Comment (MikeJohnPage) [reply R/data.R:3]: Please also make these changes to the LTLA's docs too. Don't forget to rebuild them :)

**Comment by MikeJohnPage** [comment]
Thanks for syncing with main! Made it much easier to review :)

**Comment by niuni02** [comment]
I have completed the changes you requested. Please let me know if I need to make any more changes.

**Comment by MikeJohnPage** [comment]
Thanks!

**Pull request merged**